tests/test_exports.py

```diff
@@ -50,17 +51,62 @@ def test_can_import_misc_resources() -> None:
 
 
 def test_can_import_abstract() -> None:
-    from stripe.api_resources.abstract import (
-        APIResource as APIResourceFromApiResourcesAbstract,
-    )
+    # fmt: off
+    from stripe.api_resources.abstract import APIResource as APIResourceFromAbs
+    from stripe.api_resources.abstract import SingletonAPIResource as SingletonFromAbs
+    from stripe.api_resources.abstract import CreateableAPIResource as CreateableFromAbs
+    from stripe.api_resources.abstract import UpdateableAPIResource as UpdateableFromAbs
+    from stripe.api_resources.abstract import DeletableAPIResource as DeletableFromAbs
+    from stripe.api_resources.abstract import ListableAPIResource as ListableFromAbs
+    from stripe.api_resources.abstract import SearchableAPIResource as SearchableFromAbs
+    from stripe.api_resources.abstract import VerifyMixin as VerifyMixinFromAbstract
```

Comment:
nit: I like the more explicit `...FromAbstract` naming over `FromAbs`, which sounds a little like "absolute".